project_task_analytic_tag/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 PESOL (http://pesol.es)
+#                Luis Adan Jimenez Hernandez (luis.jimenez@pesol.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Task Analytic Tag",
+    "summary": "Analytic tag in project task",
+    "version": "10.0.1.0.0",
+    "author": "PESOL",
+    "website": "http://pesol.es",
+    "category": "Analytic Accont",
+    "license": "AGPL-3",
+    "depends": [
+        'project',
+        "hr_timesheet"
```

Comment:
Hi @dreispt , it's not related, and is not a dependency.

You can use it alone, to set analytic tag on analytic lines created on timesheet.

We only talk about the other module to explain why we decided to improve analytic tags instead of migrate analytic account parent.